code/controllers/subsystem/atoms.dm

```diff
@@ -14,6 +14,10 @@ SUBSYSTEM_DEF(atoms)
 
 	var/list/BadInitializeCalls = list()
 
+
+	var/creating_atoms_list = FALSE //if this is true then ssAtoms is currently recording every atom initialized from a map template
+	var/list/created_atoms = list() //initAtom() adds the atom its creating to this list if creating_atoms_list is true
```

Comment:
```suggestion
 ///initAtom() adds the atom its creating to this list if defined and not null
	var/list/created_atoms
```

Reply:
whoops i forgot to get rid of the boolean, fixed now